unix_integration/src/idprovider/kanidm.rs

```diff
@@ -99,7 +99,7 @@ impl IdProvider for KanidmProvider {
 
         if id_key.is_none() {
             let loadable_id_key = tpm
-                .identity_key_create(machine_key, tpm::KeyAlgorithm::Ecdsa256)
+                .identity_key_create(machine_key, None, tpm::KeyAlgorithm::Ecdsa256)
```

Comment:
@Firstyear is this ok - the version of kanidm-hsm-crypto was bumped and the `PinValue` field is noted as optional.